traffic_ops/traffic_ops_golang/api/shared_handlers.go

```diff
@@ -315,25 +315,23 @@ func DeleteHandler(deleter Deleter) http.HandlerFunc {
 			HandleErr(w, r, inf.Tx.Tx, errCode, userErr, sysErr)
 			return
 		}
-		if !deleteKeyOptionExists {
```

Comment:
The diff is kind of hard to read here, all I did was remove the `if !deleteKeyOptionExists` so that keys are still set with either Deleter or OptionsDeleter.